tests/core/test_full_node_rpc.py

```diff
@@ -97,19 +97,19 @@ async def test1(two_nodes_sim_and_wallets_services, self_hostname, consensus_mod
 
         assert (await client.get_block_record_by_height(100)) is None
 
-        ph = list(blocks[-1].get_included_reward_coins())[0].puzzle_hash
+        ph = list(set(blocks[-1].get_included_reward_coins()))[0].puzzle_hash
```

Comment:
without the conversion to `set` and back to `list` in this test, it fails.

Reply:
It's like the included reward coins order is inconsistent but the coins hash consistently across Python invocations (not necessarily expected) and so the `set()` call is inducing order.  Exactly the opposite of how we generally view `set()`.

Let's at least get a comment on this.

```suggestion
        # TODO: Understand why the list(set()) is required to make this work and address it.  This shouldn't be needed.
        ph = list(set(blocks[-1].get_included_reward_coins()))[0].puzzle_hash
```